README.md

```diff
@@ -22,6 +22,13 @@ EvalML is an AutoML library which builds, optimizes, and evaluates machine learn
 pip install evalml
 ```
 
+#### Add-ons
+**Update checker** <br>
+Receive automatic notifications of new EvalML releases
+```
+python -m pip install evalml[update_checker]
```

Comment:
Mega nit-pick but elsewhere in our docs we say `pip install ...` instead of `python -m pip install ...`